README.md

```diff
@@ -53,7 +53,7 @@ Now let's move on and mount our component to the DOM. Open up "index.jsx"
 
 ```
 var React = require('react');
-var App   = require('./components/App');
+var App   = require('./components/App.jsx');
```

Comment:
Add file extension to keep consistency and clarify that App.jsx is a user created file